components/logo/LogoComponent.tsx

```diff
@@ -0,0 +1,19 @@
+import Image from "next/image";
```

Comment:
I don't think we should create a whole folder for this component

I think we can have it be `components/Logo.tsx`

Reply:
IMO it is a good practice to keep code encapsulated (still modularized) in one place specially if you are pulling out general component. It will keep it clean and more clear with approach. Also you can add custom changes and relevant component file within the folder like css, test, build, custom asset.